### DISCUSSION TITLE
TinyMCE config for superscript & subscript

### DISCUSSION THREAD

**Discussion by klickparkdominik** [pimcore/pimcore#15967] [Q&A / Support] [answered]
In the past, it was easy to set `<sub>` & `<sup>` in the WYSIWYG editor. 
These buttons are no longer available since the update.

The reason is that now the TinyMCE editor is used, which obviously does not show these buttons in the default.

Unfortunately, I have not found any documentation that is helpful to me on how to configure the TinyMCE editor used in Pimcore to display these two buttons again.

I would love it if there was a clear and practical step-by-step guide for this.

**Comment by aarongerig** [comment]
@klickparkdominik those  options are available here https://www.tiny.cloud/docs/configure/content-formatting/#generalbuilt-informats

**Reply by klickparkdominik** [reply]
Thanks!

**Comment by klickparkdominik** [comment]
We managed to show the desired buttons by editing `/public/bundles/pimcoretinymce/js/editor.js`.
Add `subscript superscript` to `const toolbar1` and `'subscript', 'superscript'` to `plugins: []`.

This might not be the best way though.

**Reply by dvesh3** [reply]
@robertSt7 any suggestions how to extend the toolbar properly?

**Reply by robertSt7** [reply, marked as answer]
Hi @klickparkdominik there are more ways to do that. Please have a look at the `Readme` here https://github.com/pimcore/pimcore/tree/11.x/bundles/TinymceBundle

**Reply by klickparkdominik** [reply]
Hi @robertSt7 thank you for pointing me into the right direction. This looks very helpful on first glance. Will definitely check it out.
